docs/troubleshooting.md

```
@@ -0,0 +1,336 @@
+  --zone=us-central1-a --format="value(networkInterfaces[0].accessConfigs[0].natIP)"
+```
+
+Compare this to the IP in `config/deploy.yml`. If they differ, either add a `google_compute_address` to your Terraform config (recommended) or manually update `deploy.yml` and DNS.
+
+## Deploy Stuck at Health Check
+
+### Symptoms
+
+`bin/kamal deploy` hangs at the kamal-proxy deploy step:
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)